src/views/Home/Home.tsx

```diff
@@ -92,27 +131,37 @@ function Home(props: Props): React.ReactElement {
     }
   };
 
-  const onInvite = async (email: string) => {
+  const invite = async (field: EmailField) => {
+    const { email } = field;
+    await validateField("email");
     try {
       await Admin.invite(email, token);
+      setInviteModal(false);
     } catch (err) {
-      // show error message
+      setMsg(err.toString());
     }
   };
 
+  const hideSnackbar = () => setMsg("");
+
+  //Sets position of the FAB depending on the platform
+  let styleFAB = {
+    right: 16,
+    position: Platform.OS === "web" ? "fixed" : "relative",
+  };
+
+  if (Platform.OS !== "web") {
+    styleFAB["top"] = window.height - safeArea.bottom;
+  } else {
+    styleFAB["bottom"] = 16;
+  }
```

Comment:
Oh wait this should match how its done on `main`, see suggestion.

```suggestion
  const styleFAB = {
    position: Platform.OS === "web" ? "fixed" : "absolute",
    margin: 16,
    right: 0,
    bottom: 0,
  };
```